src/index.html

```diff
@@ -1,14 +1,36 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>Webpack tutorial</title>
-        <script src="../dist/main.js"></script>
-    </head>
-    <body>
-        <h1>Hello webpack!</h1>
-        <p>Hello sass!</p>
-    </body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+    />
+    <title>Webpack tutorial</title>
+    <script src="../dist/main.js"></script>
+  </head>
+  <body>
+    <main>
+      <section class="do-list">
+        <div class="form-container">
+          <h1 class="list-title">Today's To Do</h1>
+          <hr class="list-line" />
+          <input
+            class="list-input"
+            placeholder="Add to your list"
+            type="text"
+          />
+          <hr class="list-line" />
+          <ul class="list-container"></ul>
+        </div>
+      </section>
```

Comment:
- [ ] It would be nice if you could add the refresh icon and the enter key icon on your to-do list design so you match yours with the original one given. Please add them so your app looks nice as the given design 👍🏽

**The Original Design**

<img width="1122" alt="Screenshot 2023-01-19 at 8 58 34 AM" src="https://user-images.githubusercontent.com/50941074/213399044-84941955-2529-4564-aec1-7b6a54b51d7e.png">


**Your Design**

<img width="1448" alt="Screenshot 2023-01-19 at 8 38 29 AM" src="https://user-images.githubusercontent.com/50941074/213398616-be726bdb-f8a8-412f-bcb3-445fb1cdde5a.png">